src/CheckIn.js

```diff
@@ -29,6 +29,7 @@ import {
   Dropdown,
 } from '@folio/stripes/components';
 import getEffectiveCallNumber from '@folio/stripes-util/lib/effectiveCallNumber';
```

Comment:
Gah! I know this isn't a PR about correct imports from stripes-util but it's just sitting there staring at me like, "Hey, whatcha gonna do about this?!?!"

Reply:
Filed https://issues.folio.org/browse/UICHKIN-175

Reply:
Oh nice find!